cmd/kubelet/app/auth_test.go

```diff
@@ -179,7 +179,7 @@ func TestAuthnWebhookRequestEncoding(t *testing.T) {
 				t.Fatalf("failed to create authentication client: %v", err)
 			}
 
-			authn, _, err := BuildAuthn(authnClient, kubeletconfig.KubeletAuthentication{
+			authn, _, err := BuildAuthn(context.TODO(), authnClient, kubeletconfig.KubeletAuthentication{
```

Comment:
In tests, create a `tCtx := ktesting.Init(t)` and use it when a `context.Context` or `logger = tCtx.Logger()` is needed.